docs/guide/intro.rst

```diff
@@ -0,0 +1,103 @@
+:orphan:
+
+.. currentmodule:: discord
+
+.. _guide_intro:
+
+Introduction
+=============
+
+Welcome to discord.py, a library for Python to aid in creating applications for Discord.
+
+Prerequisites
+--------------
+
+discord.py requires Python 3.8.0 or higher. Support for earlier versions is not provided as they have been deprecated.
+
+.. _guide_intro_installation:
+
+Installation
+-------------
+
+discord.py is available on PyPI: ::
+
+    python3 -m pip install -U discord.py
+
+If you are on Windows, you should use the following command instead: ::
+
+    py -3 -m pip install -U discord.py
+
+Voice Support
+~~~~~~~~~~~~~~
+
+Voice support (e.g. joining voice channels and playing music) is not supported by default, and can be installed by installing ``discord.py[voice]`` instead of ``discord.py``; ::
+
+    pip install -U discord.py[voice]
+
+Linux environments may need to install additional dependencies to get full voice support.
+
+For **Debian/Ubuntu** systems:
+
+.. code-block:: shell
+
+    $ sudo apt install libffi-dev libsodium-dev python3-dev
+
+For **Fedora/CentOS** systems:
+
+.. code-block:: shell
+
+    $ sudo dnf install libffi-devel libsodium-devel python3-devel
+
+For **Arch Linux** systems:
+
+.. code-block:: shell
+
+    $ pacman -Syu libsodium
+
+For other distributions, please use your package manager to find libraries for ``libffi``, ``libsodium``, and the Python 3 development headers.
+
+Windows will not need additional dependencies as they are bundled with discord.py.
+
+Virtual Environments
+~~~~~~~~~~~~~~~~~~~~~
+
+Global Python environments get cluttered with dependencies very easily - virtual environments can help separate your projects into clean, organized folders.
+Virtual environment (or "venv") is a concept introduced in Python 3.3 used to help separate project dependencies from the global Python installation, thereby not polluting
+other projects using the same Python version. They also allow you to install libraries that you may not have permission to install globally.
+
+To quickly get a virtual environment working in your project folder:
+
+1. Ensure you are in your project's root directory.
+
+    .. code-block:: shell
+
+        $ cd your_bot_source
+        $ python3 -m venv .venv
+
+    The ``.venv`` argument is the output folder of the virtual environment, this can be named anything but be sure to remember it.
+
+2. Activate the virtual environment:
+
+    .. code-block:: shell
+
+        $ source .venv/bin/activate
+
+    On Windows, use the following:
+
+    .. code-block:: pwsh
+
+        $ .\.venv\Scripts\activate
+
+3. You can then use ``pip`` and ``python`` without interfering with other projects:
+
+    .. code-block:: shell
+
+        $ pip install -U discord.py  # note the lack of ``py -3 -m`` or ``python3 -m``
+        $ python your_bot.py
+
+For a more in-depth look into virtual environments, see :doc:`py:tutorial/venv`.
```

Comment:
I personally feel like it's be nice to have this, even if it's redundant. I've been referred to the virtual environment documentation in the past, but it just feels like there's too much I'd have to learn in order to use a virtual environment, even if it is as simple as it appears to be. The guide in the commit gives a short, simple introduction anyone can use without spending a lot of time figuring out how it works.

Of course, when something goes wrong (as it inevitably will), they'll have to learn more (and this might involve them reaching out to community members), but I think that having this will be a net good. It'll get people's foot in the door and give them a chance to start learning about `venv`s.

Reply:
Sorry for the necro.

I do understand that point, and to some extent I agree. I'll leave it to the open forum of users to maybe 👍 or 👎 to let us know.
Unless there's an executive decision made on it.

Reply:
I personally don't mind it, I realize I haven't contributed much to these docs but I always see the use in fully fleshed-out introductions.

Given that this is specifically called out in the "intro" section I feel we should definitely be giving as much help as we can